From 6f78c2db4a4f507feaf21ecc85e92511991edb56 Mon Sep 17 00:00:00 2001
From: Tony Cheng <tony.cheng@amd.com>
Date: Fri, 26 Aug 2016 00:17:38 -0400
Subject: [PATCH 0945/1722] drm/amd/dal: remove get_din_connector_info.

This is for CV which wasn't supported in dal2 either.

Change-Id: Idceb9b6209f88261dec94ab285156069eeee4a91
Signed-off-by: Jordan Lazare <Jordan.Lazare@amd.com>
Acked-by: Harry Wentland <harry.wentland@amd.com>
Signed-off-by: Kalyan Alle <kalyan.alle@amd.com>
---
 drivers/gpu/drm/amd/dal/dc/bios/bios_parser.c | 2 --
 drivers/gpu/drm/amd/dal/dc/dc_bios_types.h    | 4 ----
 2 files changed, 6 deletions(-)

diff --git a/drivers/gpu/drm/amd/dal/dc/bios/bios_parser.c b/drivers/gpu/drm/amd/dal/dc/bios/bios_parser.c
index 60b5ceb..c8ea7d6 100644
--- a/drivers/gpu/drm/amd/dal/dc/bios/bios_parser.c
+++ b/drivers/gpu/drm/amd/dal/dc/bios/bios_parser.c
@@ -4402,8 +4402,6 @@ static const struct dc_vbios_funcs vbios_funcs = {
 
         .get_encoder_cap_info = bios_parser_get_encoder_cap_info,
 
-        .get_din_connector_info = bios_parser_get_din_connector_info,
-
         .is_accelerated_mode = bios_parser_is_accelerated_mode,
 
         .set_scratch_critical_state = bios_parser_set_scratch_critical_state,
diff --git a/drivers/gpu/drm/amd/dal/dc/dc_bios_types.h b/drivers/gpu/drm/amd/dal/dc/dc_bios_types.h
index 1c437d3..e0e7144 100644
--- a/drivers/gpu/drm/amd/dal/dc/dc_bios_types.h
+++ b/drivers/gpu/drm/amd/dal/dc/dc_bios_types.h
@@ -110,10 +110,6 @@ struct dc_vbios_funcs {
 		struct dc_bios *bios,
 		struct graphics_object_id object_id,
 		struct bp_encoder_cap_info *info);
-	enum bp_result (*get_din_connector_info)(
-		struct dc_bios *bios,
-		struct graphics_object_id id,
-		struct din_connector_info *info);
 
 	bool (*is_lid_status_changed)(
 		struct dc_bios *bios);
-- 
2.7.4

